syntax = "proto2";

package capsuleDBSerialization;

import "src/core/kvs_payload.proto";

message CapsuleBlock{
  optional int32 level = 1;
  optional string startKey = 2;
  optional string endKey = 3;

  repeated kvs_payload kvPairs = 4;
}
